deploy/upgrade_db_server.yml

```diff
@@ -34,6 +34,14 @@
   roles:
     - role: postgresql-server
   post_tasks:
+    - name: Ensure PG is started
+      service:
+        name: "{{ pp_servicename }}"
+        state: started
+        enabled: yes
+      tags:
+        - postgresql-server
```

Comment:
@rioh This is actually already done on line 51 below, it is just missing the `enabled: yes` part, could you add that in and remove this block?